src/actions/signupAction.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+const signupHandle = async (signupData) => {
+  try {
+    const res = await axios.post(`http://localhost:5000/api/v1/users/signup`, {
```

Comment:
Great job so far, but however I think you should try to use Heroku URL instead of the Local one.